src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,62 @@
+$heigt-watch: 500px;
+$width-watch: 500px;
+
+.stop-watch {
+  width: $width-watch;
+  height: $heigt-watch;
+  background-color: #b8a3a3;
+  margin: 50px auto;
+  position: relative;
+  border-radius: 50%;
+  border: 10px dotted black;
+
+  &__minute {
+    position: absolute;
+    height: $heigt-watch/60;
+    width: $width-watch/2.5;
+    background-color: #993636;
+    top: 50%;
+    left: 50%;
+    rotate: -90deg;
+    transform-origin: 2%;
+    translate: -2% -50%;
+    border-radius: 20px;
+    animation: run-watch 3600s linear infinite;
+  }
+
+  &__second {
+    position: absolute;
+    height: $heigt-watch/60;
+    width: $width-watch/2;
+    background-color: #10d0e9a4;
+    top: 50%;
+    left: 50%;
+    rotate: -90deg;
+    transform-origin: 2%;
+    translate: -2% -50%;
+    border-radius: 20px;
+    animation: run-watch 60s linear infinite;
+  }
```

Comment:
you have such a lot of repeatable styles here. 
Use either `mixin` or common style like `stop-watch__hand` where you will collect common styles and modifiers `--minute `and `--second` to apply difference

Example: 

```
.hand {
  {{common_styles}}
 
  &--second {
    {{styles_only_for_second}}
  } 
  
  &--minute {
    {{styles_only_for_minute}}
  } 
}
```